### PR TITLE
Slack interg improve

### PR DESCRIPTION
**Pull request by karakum**
1. Fix prev commit "make unsets when other fieldParams are set logic more compact" - impossible to disable unset of `text` param. To turn ON unset of `text` just add `unset_text: true` to config.

2. Real hostname in **Host** attachment block.

3. Add customization of attachment color & title by adding to slack config `attachmentCustom`:
```
  slack:
    token: xoxp...
    ...
    attachmentCustom:
      color: '#a94442'
#      title: 'Деплой выполнен'
```

### CODE REVIEW

File: slack.php [hunk 1]
```diff
@@ -87,6 +86,9 @@
     }
 
     $config = array_merge($defaultConfig, (array) $newConfig);
+    if(isset($newConfig['attachmentCustom'])){
```
Comment (antonmedv) [inline slack.php]: Please, fix code style

Comment (karakum) [reply slack.php]: What does you mean? 
`attachment-custom` ?
`attachment_custom` ?

Comment (antonmedv) [reply slack.php]: Spaces: ` if (isset($newConfig['attachmentCustom'])) {`

Comment (karakum) [reply slack.php]: @antonmedv fixed

**Comment by antonmedv** [comment]
On weekends

**Comment by karakum** [comment]
No release yet

**Pull request merged**